bcheng-ms/azure-search-openai-demo
# add-load-data

## Purpose
Load and index data

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Add data into a branch.
Run the workflow dispatch
Check to see if the data was loaded into the storage account
Check to see if the search index was created.
